cli/cook/querying.py

```diff
@@ -243,19 +243,21 @@ def __get_latest_instance(job):
     raise CookRetriableException(f'Job {job["uuid"]} currently has no instances.')
 
 
-def query_unique_and_run(clusters, entity_ref, command_fn, wait=False):
+def query_unique_and_run(clusters, entity_ref, command_fn, wait=False, do_not_resove_sandbox_directory=False):
```

Comment:
Seems easier to understand/read to have this be inverted - `resolve_sandbox_directory=True`

Reply:
ok